### PR TITLE
Svg sprite support

### PR DESCRIPTION
**Pull request by jonerd**
https://github.com/biotope/biotope-build/issues/46

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "gulp-sass": "3.1.0",
     "gulp-sass-lint": "1.3.4",
     "gulp-size": "2.1.0",
-    "gulp-sourcemaps": "2.6.1",
+    "gulp-sourcemaps": "2.6.4",
     "gulp-svg-sprite": "^1.3.7",
```
Comment (timomayer) [inline package.json]: should be updated to 1.4.0

Comment (jonerd) [reply package.json]: Update is done

**Comment by timomayer** [comment]
great, thanks!!

**Comment by SheepFromHeaven** [comment]
@jonerd please resolve conflicts

**Comment by jonerd** [comment]
@SheepFromHeaven conflicts resolved

**Pull request merged**